CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.9.0 / 2019-12-20
+
+After a testing period of 7 days, there were no additional bugs found or features introduced.
```

Comment:
Did we want to say something about this being the last release in the CHANGELOG or is this the wrong place for it?

Reply:
I don't think we need it in the `CHANGELOG` IMO.